invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/ItemsSearch/components/ResultsList/ResultsList.js

```diff
@@ -8,14 +9,16 @@ export class ResultsList extends Component {
     this.viewDetailsClickHandler = this.props.viewDetailsClickHandler;
   }
 
+  _getFormattedDate = d => (d ? toString(fromISO(d)) : '');
```

Comment:
there was a function created in /common/api/date to avoid copying this line this should be fixed when https://github.com/inveniosoftware/invenio-app-ils/issues/123 will be done